dataprofiler/reports/graphs.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+import math
+import warnings
+
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+
+def plot_histograms(profiler, columns=None):
+    """
+    Take a input of StructuredProfiler class and a list of specified column
+    names and then plots the histograms of those that are int or float
+    columns.
+
+    :param profiler: StructuredProfiler variable
+    :param columns: list of column names to be plotted
+    :type profiler: StructuredProfiler
+    :type columns: list
+    :return: returns fig
+    :rtype: fig
+    """
+
+    # get all inds to graph, raise error if user specified doesn't exist (part 1)
```

Comment:
I would remove the `(part x)` comments, personally, I think just listing out the steps is sufficient indication of a "step" of computation. Not a huge issue though, won't block because of it.